client/lib/directly/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { loadScript } from 'lib/load-script';
+
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	id: config( 'directly_rtm_widget_id' ),
+	displayAskQuestion: false
+};
+const DIRECTLY_RTM_SCRIPT_URL = 'http://widgets.wp.com/directly/embed.js';
+const DIRECTLY_ASSETS_BASE_URL = 'https://www.directly.com';
+let initializationPromise;
+
+function insertDirectlyDOM() {
+	// Directly gathers the base URL for its assets by inspecting the src attribute
+	// of a DOM element with id="directlyRTMScript". Since we're not using the standard
+	// integration, we need to fake it by adding a dummy DOM object with those attributes.
+	const d = document.createElement( 'div' );
+	d.id = 'directlyRTMScript';
+	d.src = DIRECTLY_ASSETS_BASE_URL;
+	document.body.appendChild( d );
+}
+
+function executeDirectlyCommand( ...args ) {
+	// Havoc is wreaked if you try to execute Directly commands before initialization.
+	// If initialization hasn't started, ignore the command.
+	if ( typeof initializationPromise === 'undefined' ) {
+		return;
+	}
+	// If initialization has started, add the command to the promise chain to execute
+	// as soon as initialization has finished.
+	return initializationPromise.then( () => {
+		window.DirectlyRTM( ...args );
+	} );
```

Comment:
fun fact! no need for the curlies here as all we're doing is evaluating an expression (in this case that's the return vape from the function call or probably actually `undefined`)

```js
return initializationPromise.then( () => window.directlyRTM( ...args ) );
```

if you prefer to inline the thing that's possible too

```js
const executeDirectlyCommand = ( ...args ) =>
	'undefined' !== typeof initializationPromise
		? initializationPromise.then( () => window.directlyRTM( ...args ) )
		: undefined;
```